addons/sale_pdf_quote_builder/models/sale_order.py

```diff
@@ -19,16 +19,19 @@ def _default_quotation_document_ids(self):
         string="Available Quotation Documents",
         comodel_name='quotation.document',
         compute='_compute_available_quotation_document_ids',
+        groups='sales_team.group_sale_salesman',
```

Comment:
Mhm, in my opinion, if you have access to the SO, you should see (and have access to) the quote builder logic too (same comment for the other fields in this file).

Reply:
@Demesmaeker 

> For the default (or selected) documents, I don't think this is right. I could agree for the available_quotation_document_ids, but if you have access to the SO, then you should have access to the selected documents that are attached to it IMHO. Especially as they'll be in the mail sent to the customer, and thus present in the log notes. 🤔

Updated here and for all fields.